pixl_core/src/core/omop.py

```diff
@@ -65,24 +71,52 @@ def copy_to_exports(
             raise FileNotFoundError(msg)
 
         # Make directory for exports if they don't exist
-        project_slug, extract_time_slug = self._get_slugs(project_name, extract_datetime)
-        export_base = self.export_dir / project_slug
-        public_output = OmopExtract._mkdir(
-            export_base / "all_extracts" / "omop" / extract_time_slug / "public"
-        )
-        logger.info("Copying public parquet files from %s to %s", omop_dir, public_output)
+        ParquetExport._mkdir(self.public_output)
+        logger.info("Copying public parquet files from %s to %s", omop_dir, self.public_output)
 
         # Copy extract files, overwriting if it exists
-        shutil.copytree(public_input, public_output, dirs_exist_ok=True)
+        shutil.copytree(public_input, self.public_output, dirs_exist_ok=True)
         # Make the latest export dir if it doesn't exist
-        latest_parent_dir = self._mkdir(export_base / "latest" / "omop")
+
+        self._mkdir(self.latest_parent_dir)
         # Symlink this extract to the latest directory
-        latest_public = latest_parent_dir / "public"
+        latest_public = self.latest_parent_dir / "public"
         if latest_public.exists():
             latest_public.unlink()
 
-        latest_public.symlink_to(public_output, target_is_directory=True)
-        return project_slug
+        latest_public.symlink_to(self.public_output, target_is_directory=True)
+        return self.project_slug
+
+    def export_radiology(self, anon_data: list[tuple[Any, ...]]) -> pathlib.Path:
+        """Export radiology reports to parquet file"""
+        # The parquet file should have the following columns:
+        # - De-IDed report text
+        # - procedure_occurrence_id (aka. EHR imaging identifier)
+        # - DICOM link (ie. the hashing API's response for {patient mrn}{accession number})
+
+        self._mkdir(self.radiology_output)
+        parquet_file = self.radiology_output / "radiology.parquet"
+
+        parquet_header_names = ["image_identifier", "procedure_occurrence_id", "image_report"]
+        if anon_data and len(parquet_header_names) != len(anon_data[0]):
+            err_str = (
+                f"Passed in data must have expected number of columns "
+                f"(got {len(anon_data[0])}, expected {len(parquet_header_names)})"
+            )
+            raise ValueError(err_str)
+        export_df = pd.DataFrame(anon_data, columns=parquet_header_names)
```

Comment:
Is there a reason you went for making the dataframe here, and not being passed a dataframe that's been built directly from the query in the `ehr-api`? 

Feels like if the query changed (e.g. changing columns) we'd have to change it here, rather than changing it all in one place

Reply:
Yes, I think it's better to do it all together like that, so I'm moving it. Also removed the validation as it turns out pandas does that anyway. Only slight thing is that we now have some parquet extract-specific logic (the column names) in the `_databases.py` module, but eh.